docker/Dockerfile

```diff
@@ -55,11 +55,12 @@ RUN chmod a+x /usr/local/bin/build-farmOS.sh
 # Change the ownership of the sites directory and copy the farmOS codebase into /opt/drupal.
 RUN mkdir /var/farmOS \
   && /usr/local/bin/build-farmOS.sh --no-dev \
-  && chown -R www-data:www-data /var/farmOS/web/sites \
+  && chown -R www-data:www-data /var/farmOS/web/sites /var/www/html/sites\
   && rm -r /opt/drupal && cp -rp /var/farmOS /opt/drupal
 
 # Set the entrypoint.
```

Comment:
Can we split this block up a bit more with additional comments, now that we are doing more than just "setting the entrypoint"?

Perhaps:

```
# Copy the entrypoint script and make it executable.
COPY docker-entrypoint.sh /usr/local/bin/
RUN chmod u+x /usr/local/bin/docker-entrypoint.sh

# Run as www-data user.
USER www-data

# Set the entrypoint and command.
ENTRYPOINT ["docker-entrypoint.sh"]
CMD ["apache2-foreground"]
```

Reply:
Sounds good, I've made this change in an extra commit.